client/src/components/new-request/submit-request/submission-tabs.vue

```diff
@@ -37,48 +36,61 @@
 </template>
 
 <script lang="ts">
+import { Component, Vue, Watch } from 'vue-property-decorator'
+import NewReqModule from '@/store/new-request-module'
+import MainContainer from '../main-container.vue'
+import EntityCannotBeAutoAnalyzed from './entity-cannot-be-auto-analyzed.vue'
+import NamesCapture from '../../common/names-capture.vue'
 import ApplicantInfo1 from '../../common/applicant-info-1.vue'
 import ApplicantInfo2 from '../../common/applicant-info-2.vue'
 import ApplicantInfo3 from '../../common/applicant-info-3.vue'
-import EntityCannotBeAutoAnalyzed from './entity-cannot-be-auto-analyzed.vue'
 import InvalidActionMessage from '../../existing-request/invalid-action-message.vue'
-import MainContainer from '../main-container.vue'
-import NamesCapture from '../../common/names-capture.vue'
-import newReqModule from '@/store/new-request-module'
 import Timeout from './timeout.vue'
-import { Component, Vue } from 'vue-property-decorator'
 
+/**
+ * This is the component container for a new submission.
+ */
 @Component({
   components: {
+    MainContainer,
+    EntityCannotBeAutoAnalyzed,
+    NamesCapture,
     ApplicantInfo1,
     ApplicantInfo2,
     ApplicantInfo3,
-    EntityCannotBeAutoAnalyzed,
     InvalidActionMessage,
-    MainContainer,
-    NamesCapture,
     Timeout
   }
 })
 export default class SubmissionTabs extends Vue {
-  mounted () {
+  private mounted (): void {
     let link = document.createElement('link')
     link.setAttribute('href', 'https://cdnjs.cloudflare.com/ajax/libs/open-iconic/1.1.1/font/css/open-iconic.min.css')
     link.setAttribute('rel', 'stylesheet')
     document.head.appendChild(link)
   }
-  get actingOnOwnBehalf () {
-    return newReqModule.actingOnOwnBehalf
+
+  private get actingOnOwnBehalf (): boolean {
+    return NewReqModule.actingOnOwnBehalf
   }
-  get submissionTabNumber () {
-    return newReqModule.submissionTabNumber
-  }2
-  set submissionTabNumber (value) {
-    newReqModule.mutateSubmissionTabNumber(value)
+
+  private get submissionModeHeader (): string {
+    // safety check
+    if (NewReqModule.editMode) return ''
+
+    if (this.submissionTabNumber === 1) {
+      return 'Submit a Name Request for Review'
+    } else {
+      return 'Submission Details'
+    }
```

Comment:
This is now almost identical code as in ExistingRequestEdit (consistency and symmetry).